04 Strategy Library/230 Fama French Five Factors/02 Fama French Five-Factor Model.html

```diff
@@ -0,0 +1,17 @@
+<p>
+    The Fama French five-factor model was proposed in 2014 and is adapted from the Fama French three-factor model (Fama and French, 2015). It builds upon the dividend discount model which states that the value of stocks today is dependent upon future dividends.
+    Fama and French add two factors, investment and profitability, to the dividend discount model to better capture the relationship between risk and return.
+    The model is as follows
+</p>
+\[ R = \alpha + \beta_m MKT + \beta_s SMB + \beta_h HML + \beta_r RMW + \beta_c CMA\]
+
+<p>
+where
+</p>
+<ul>
+    <li>MKT is the excess return of the market. It is the return on the value-weight market portfolio.</li>
+    <li>SMB is the return on a diversified portfolio of small stocks minus the return on a diversified portfolio of big stocks.</li>
+    <li>HML is the difference between the returns on diversified portfolios of high and low B/M stocks.</li>
```

Comment:
Use "HML is the difference between the returns on diversified portfolios of stocks with high and low Book-to-Market ratios." since not all readers will know what "B/M" means